MetaMask/metamask-mobile
# fix: update to unable to find conversion rate when failing to get conversion rate

## **Description**

Currently we are displaying "unable to load balance" whenever the app is not able to get conversion rates for an asset. To be more accurate we should update the message to say "unable to find conversion rate" for this scenario.

## **Related issues**

Fixes:  [#8151 ](https://github.com/MetaMask/metamask-mobile/issues/8151)

## **Manual testing steps**

1. Go to the wallet view
2. import uniswap token in sepolia testnet
3. app is not able to retrieve conversion rate
4. "unable to load conversion rates" should be displayed (FAILED: "unable to load balance" is displayed)

## **Screenshots/Recordings**



### **Before**

<img width="466" alt="before" src="https://github.com/MetaMask/metamask-mobile/assets/26223211/14e911ce-c428-423e-9765-4474452f5e42">


### **After**
<img width="470" alt="after" src="https://github.com/MetaMask/metamask-mobile/assets/26223211/32c31e52-836f-4e3a-b2a7-c03c797fae42">


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
